### PR TITLE
Test to automate RHSTOR 4474: On-premise deployments of ODF, where we have 5 or more nodes,racks or rooms  add the ability to configure 5 MONs

### PR DESCRIPTION
**Pull request by pintojoy**

### CODE REVIEW

**Comment by pintojoy** [comment]
Verified the PR using the job https://url.corp.redhat.com/4e6817f.

File: tests/functional/z_cluster/test_five_mon.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+    """
+    ret = pods.wait_for_resource(
+        condition=constants.STATUS_RUNNING,
+        selector="app=rook-ceph-mon",
+        resource_count=5,
+        timeout=660,
+    )
+    log.info(f"Waited for all mon pods to come up and running {ret}")
+    return ret
```
Comment (ramkiperiy) [inline tests/functional/z_cluster/test_five_mon.py:38]: There is no existing library file to validate running mon pod? if not could you please move this to library file

File: tests/functional/z_cluster/test_five_mon.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+from ocs_ci.framework import config
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.utility import prometheus
+from ocs_ci.ocs.resources.pod import verify_mon_pod_running
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+@ignore_leftovers
```
Comment (ramkiperiy) [inline tests/functional/z_cluster/test_five_mon.py:24]: any specific reason for leftover marker?

File: tests/functional/z_cluster/test_five_mon.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+@skipif_less_than_five_workers
+@skipif_ocs_version("<4.15")
+class TestFiveMonInCluster(ManageTest):
+    def test_five_mon_pod_in_cluster(self, threading_lock):
```
Comment (ramkiperiy) [inline tests/functional/z_cluster/test_five_mon.py:28]: Update the TC name something like test to scale 5mon pod

File: tests/functional/z_cluster/test_five_mon.py (hunk 2 above)
Comment (pintojoy) [reply tests/functional/z_cluster/test_five_mon.py:24]: As the test updates mon count to five and we do not have option to scale down for now, added ignore leftover label to address teardown failures

**Comment by pintojoy** [comment]
Was facing PR check failures, hence created a new PR at https://github.com/red-hat-storage/ocs-ci/pull/9980

**Pull request closed without merging**